api/server/server.go

```diff
@@ -717,15 +689,14 @@ func (s *Server) postImagesCreate(version version.Version, w http.ResponseWriter
 		}
 
 		imagePullConfig := &graph.ImagePullConfig{
-			Parallel:    version.GreaterThan("1.3"),
+			Parallel:    true,
```

Comment:
in fact you can remove Parallel altogether, it's equivalent to true wherever it's referenced.